src/components/Editor/SearchBar.js

```diff
@@ -305,6 +179,11 @@ class SearchBar extends Component {
     this.traverseResults(e, e.shiftKey);
     e.preventDefault();
   }
+
+  getInputRef(c) {
+    this.searchInput = c;
```

Comment:
this change doesnt make much sense to me, because this allows us to highjack the search and allows us to set the search field in two ways, via props and through setSearchValue. 

we should instead have local state and props, props being the state that is sent to the search worker.